selfdrive/car/ford/carstate.py

```diff
@@ -88,8 +88,8 @@ def update(self, cp, cp_cam):
   @staticmethod
   def parse_gear_shifter(gear: str) -> car.CarState.GearShifter:
     d: Dict[str, car.CarState.GearShifter] = {
-        'Park': GearShifter.park, 'Reverse': GearShifter.reverse, 'Neutral': GearShifter.neutral,
-        'Manual': GearShifter.manumatic, 'Drive': GearShifter.drive,
+        'PARK': GearShifter.park, 'REVERSE': GearShifter.reverse, 'NEUTRAL': GearShifter.neutral,
+        'MANUAL': GearShifter.manumatic, 'DRIVE': GearShifter.drive,
```

Comment:
hmm, we can just add this onto the dict in `selfdrive/car/interfaces.py`, then do `d.get(gear.upper(), GearShifter.unknown)`

Reply:
sounds good :)